RPM/scripts/postinst/kaltura-db-config.sh

```diff
@@ -26,6 +26,83 @@ if [ "$#" -lt 4 ];then
 	exit 1
 fi
 
+
+MYSQL_HOST=$1
```

Comment:
I think you better use the same methodology as the rest of the config.sh files.
Try to get these arguments from system.ini, from answers file, or from user input.